core/utils.h

```diff
@@ -26,6 +26,16 @@ lwm2m_media_type_t utils_convertMediaType(coap_content_type_t type);
 uint8_t utils_getResponseFormat(uint8_t accept_num, const uint16_t *accept, int numData, const lwm2m_data_t *dataP,
                                 bool singleResource, lwm2m_media_type_t *format);
 int utils_isAltPathValid(const char *altPath);
+/** Copy a string.
+ *
+ * Use utils_strncpy if possible!
+ * This is less safe than utils_strncpy.
+ *
+ * @param buffer The destination buffer
+ * @param length The mex. size of the destination buffer
```

Comment:
a) Typo
b) "Max size of teh buffer" feels like a wrong description.

Maybe this?
```suggestion
 * @param length The max number of bytes to be written to in the destination buffer
```